.eslintrc.js

```diff
@@ -15,6 +15,7 @@ module.exports = {
   rules: {
     "consistent-return": "off",
     "no-warning-comments": "off",
+    "max-len": ["error", { "code": 120 }],
```

Comment:
NIT
```suggestion
    "max-len": ["warning", { "code": 120 }],
```

Reply:
Wouldn't it be better to have an inline `eslint-disable` comment if we want to ignore a particular line?

Reply:
Does using warning prevent using it? I didn't know that. If that's the case, ignore my comment.

Reply:
It doesn't prevent using it, but if it's `warn` then people can push violating lines to CI and it will pass. If we use `error` then they need to be explicit about it by adding a `eslint-disable` disable comment.

Checking grokt, I see 8 projects using `warn` and 42 using `error`.